app/controllers/api/v1/rents_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
       def create
         rent = Rent.create!(rent_params)
+        RentMailer.with(rent: rent).new_rent_email.deliver_later
```

Comment:
Becarful, here you are using redis and sidekiq to handle your mailer. For that resaon you can not pass object as parameters. So please pass the rent_id and retreive it in the mailer ;)

Reply:
I fixed that as you requested. 
But I'm pretty sure that would work on AWS or any other infrastructure since when the parameter is an active record object rails use GlobalID::Identification.

https://apidock.com/rails/ActiveJob/Enqueuing/ClassMethods/perform_later
https://github.com/rails/globalid#usage

Reply:
Ok, you got a point, but I would like to test it in the future to be sur :)
Because last time I do this I had error.